src/style.css

```diff
@@ -1 +1,46 @@
-/* add styles here */
+body,
+html {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). In this case, you're setting the margin and padding for both `body` and `html` to 0, which is acceptable. However, ensure that this is the only place where tag names are used for styling, and use classes for styling all other elements.